_posts/2018-09-25-pulp-3-adopts-dynaconf.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Pulp 3 adopts dynaconf
+author: Dennis Kliban
+---
+Pulp 3 now uses [dyanconf](https://dynaconf.readthedocs.io/en/latest/) to manage its settings.
+dynaconf allows storing settings in
+[multiple file formats](https://dynaconf.readthedocs.io/en/latest/guides/examples.html).
+By default Pulp looks for settings in ``/etc/pulp/settings.py``. An alternate location for the
+settings file is specified by setting the ``PULP_SETTINGS`` environment variable. Each of the
```

Comment:
To specify another settings file location, set the \`\`PULP_SETTINGS\`\` environment variable.